sql/moz-fx-data-marketing-prod/ga_derived/blogs_empty_check_v1/query.sql

```diff
@@ -1,6 +1,6 @@
 SELECT
   IF(COUNT(*) = 0, ERROR(CONCAT('No data for given date ', @submission_date)), 1)
 FROM
-  `ga-mozilla-org-prod-001.66602784.ga_sessions_*`
+  `moz-fx-data-marketing-prod.66602784.ga_sessions_*`
```

Comment:
```suggestion
  66602784.ga_sessions_*
```

According to style guidelines for this repo, we should avoid fully-qualified table references except where referencing a different project from the project in which the query runs.

Reply:
This is needed for dryrun since the dryrun function uses shared-prod as the default project.  I made a issue https://github.com/mozilla/bigquery-etl/issues/1520 but I haven't had a chance to address it.

Reply:
I forgot about that issue. Understood.